cibyl/models/openstack/node.py

```diff
@@ -27,4 +28,45 @@ class Node(Model):
     """
     # To be implemented in future PR
 
-    API = {}
+    API = {
+        'name': {
+            'attr_type': str,
+            'arguments': [Argument(name='--name', arg_type=str,
+                          description="Node name")]
+        },
+        'node_id': {
```

Comment:
Probably not needed. I believe the name would be the unique ID as one deployment can't have two nodes with the same name